integrationExamples/videoModule/index.html

```diff
@@ -0,0 +1,129 @@
+<!--
+
+-->
+
+<html>
+
+<head>
+    <script async src="../../build/dev/prebid.js"></script>
+    <script src="https://cdn.jwplayer.com/libraries/l5MchIxB.js"></script>
+
+    <script>
+      var pbjs = pbjs || {};
+      pbjs.que = pbjs.que || [];
+
+      var adUnits = [{
+        code: 'div-gpt-ad-51545-0',
+        mediaTypes: {
+          video: {}
+        },
+        video: {
+          divId: 'vid-div-1',
+          adServer: {
+            vendorCode: "gam",
+            params: {
+              iu: '/19968336/prebid_cache_video_adunit',
+              cust_params: {
+                section: 'blog',
+                anotherKey: 'anotherValue'
+              },
+              output: 'vast'
+            }
+          },
+        },
+
+        // Replace this object to test a new Adapter!
+        bids: [{
+          bidder: 'appnexus',
+          params: {
+            placementId: 13144370
+          }
+        }]
+      }];
+
+
+      pbjs.que.push(function () {
+        pbjs.setConfig({
+          video: {
+            providers: [{
+              divId: 'vid-div-1',
+              vendorCode: 1,
+              playerConfig: {
+                licenseKey: 'huJTAG1dRVvvE1bT4Ng2DEnZ2ogCSgwTLwQBp8Cp9BDyAgkKC7uLuw==',
+                params: {
+                  vendorConfig: {
+                    "file": "http://content.bitsontherun.com/videos/3XnJSIm4-52qL9xLP.mp4",
+                    mediaid: 'd9J2zcaA',
+                    advertising: { client: 'googima' }
+                    }
+                }
+              },
+              adServer: {
+                vendorCode: "gam"
+              }
+            }, {
+              divId: 'vid-div-2',
+              vendorCode: 1,
+              playerConfig: {
+                licenseKey: 'huJTAG1dRVvvE1bT4Ng2DEnZ2ogCSgwTLwQBp8Cp9BDyAgkKC7uLuw==',
+                params: {
+                  vendorConfig: {
+                    "file": "http://content.bitsontherun.com/videos/bkaovAYt-DZ7jSYgM.mp4",
+                    mediaid: 'bkaovAYt',
+                    advertising: { client: 'googima' }
+                  }
+                }
+              },
+              adServer: {
+                vendorCode: "gam"
+              }
+            }]
+          }
+        });
+
+        pbjs.onEvent('setupComplete', () => {
```

Comment:
Should these events have 'video' in their name? they share the namespace with all of the rest of prebid.

Reply:
I'm not against namespacing but the cost would be longer strings

Reply:
since in the code you have a separation between video and prebid events, you could just prepend 'video' (or similar) when you propagate from one to the other - the string would be longer but the code hardly would. 

This may be another topic to bring up in the PMC; once these names are set we're stuck with them.

Reply:
What is PMC ?